themes/index.js

```diff
@@ -167,6 +167,12 @@ const themes = {
     text_color: "24292e",
     bg_color: "ffffff",
   },
+  "color-blind-friendly-dark": {
```

Comment:
can we change the name of the theme to something shorter? this is too long. 

"a11y-dark", "vision-friendly-dark" ?

Reply:
I have changed the theme name to `vision-friendly-dark`